app/src/test/java/com/keylesspalace/tusky/network/InstanceSwitchAuthInterceptorTest.kt

```diff
@@ -0,0 +1,142 @@
+package com.keylesspalace.tusky.network
+
+import com.keylesspalace.tusky.db.AccountEntity
+import com.keylesspalace.tusky.db.AccountManager
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNull
+import org.junit.Before
+import org.junit.Test
+import org.mockito.kotlin.doAnswer
+import org.mockito.kotlin.mock
+
+class InstanceSwitchAuthInterceptorTest {
+
+    private val mockWebServer = MockWebServer()
+
+    @Before
+    fun setup() {
+        mockWebServer.start(80)
```

Comment:
Isn't 80 a privileged port?
They also say
>Automated tests should always use port 0 to avoid flakiness when a specific port is unavailable.

Reply:
yes it is the standard http port, using a different one require changes to the InstanceSwitchAuthInterceptor though

Reply:
With some creativity I got it to work 👏 
Thank you!